wp-content/themes/inf_theme/admin/class-media.php

```diff
@@ -51,10 +28,10 @@ public function add_theme_support() {
   /**
    * Create new image sizes
    *
-   * @since 2.0.0
+   * @since 1.0.0
    */
   public function add_custom_image_sizes() {
-    add_image_size( 'full_width', 9999, 9999, false );
+    add_image_size( 'full-width', 1440, 9999, false );
```

Comment:
Wouldn't this limit the width to 1440px on full-width images? What if someone wants to put 2000px image in the background or something like that?

Reply:
I agree that we should encourage optimization, but limiting this could cause issues opened with: why is my image not full size? :D Setting it to `9999` basically means, it will be whatever you put it (unless you put 30k px image xD)

Reply:
if this is an issue then I will remove it because this actually does nothing. if you upload the image it will resize it to its max size which is its uploaded size.  It is the same as full.

> the_post_thumbnail( 'full' );          // Full resolution (original size uploaded)